includes/content-distribution/class-cli.php

```diff
@@ -188,57 +188,57 @@ public function cmd_distributor_migrate( array $pos_args, array $assoc_args ): v
 		if ( isset( $assoc_args['all'] ) ) {
 			$strict = isset( $assoc_args['strict'] );
 
-			$post_ids = Distributor_Migrator::get_posts_with_distributor_subscriptions();
```

Comment:
`Distributor_Migrator::get_posts_with_distributor_subscriptions` can be removed if not in use anymore